js/kucoin.js

```diff
@@ -786,6 +786,18 @@ module.exports = class kucoin extends Exchange {
         return (type === 'contract') || (type === 'future') || (type === 'futures'); // * (type === 'futures') deprecated, use (type === 'future')
     }
 
+    isMarginMethod (methodName, params) {
+        //
+        // Helper
```

Comment:
These docstrings don't follow our spec and since this is an internal/helper method I don't think they're needed

Reply:
ok, i have removed it.